clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -227,8 +275,8 @@ public void testPollEnsureEmptyPendingRequestAfterPoll() {
         CommitRequestManager commitRequestManager = create(true, 100);
         when(coordinatorRequestManager.coordinator()).thenReturn(Optional.of(mockedNode));
         Map<TopicPartition, OffsetAndMetadata> offsets = Collections.singletonMap(
-            new TopicPartition("topic", 1),
-            new OffsetAndMetadata(0));
+                new TopicPartition("topic", 1),
```

Comment:
could you undo these changes that aren't related to the pr?

Reply:
When I try to change it back, it says that the file is the same. I copied the parts of the test that I did not change from a different branch but Git is saying the files are the same still. Not too sure why it showed I made indentations, they are not showing up on my end in Intellij.

Reply:
i wonder if intellij's auto indentation automatically changes the indentation when copying the content over.  double check the indentation for these files they should be there.  it could be easier to use a text editor like vim/emac to edit out these unintended changes.